Src/BootCamp.Chapter/Shop.cs

```diff
@@ -1,28 +1,22 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Collections.Generic;
+
+namespace BootCamp.Chapter
 {
     public class Shop
     {
-        private decimal _money;
-        public decimal GetMoney()
-        {
-            return _money;
-        }
-
-        private Inventory _inventory;
+        public Inventory Inventory { get; private set; }
```

Comment:
Remove private setter - you never use it.

Reply:
Removed unused private setter for the Inventory component in Shop.